### ISSUE TITLE
ERROR! Download URL for lens is empty. Skipping.

### ISSUE THREAD

**Issue by falko** [wimpysworld/deb-get#708]
## Reporting a bug 🐛

The installation of Lens fails

### Expected behavior

Lens gets installed.

### Actual behavior

```
[!] ERROR! Download URL for lens is empty. Skipping.
```

### Steps to reproduce

Describe the exact steps required to reproduce the issue.
```bash
deb-get install lens
```

### System information

Run the following a paste the output below:

#### /etc/os-release

```bash
cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
```

PRETTY_NAME="Ubuntu 22.04.1 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.1 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy

#### /etc/lsb-release

```bash
cat /etc/lsb-release
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.1 LTS"

#### deb-get version

```bash
deb-get version
```
0.3.6

### deb-get update

```bash
deb-get update
```
Get:1 https://download.docker.com/linux/ubuntu impish InRelease [48,9 kB]
Hit:2 https://linux.teamviewer.com/deb stable InRelease
Hit:3 https://dl.winehq.org/wine-builds/ubuntu impish InRelease
Hit:4 https://updates.signal.org/desktop/apt xenial InRelease
Hit:5 https://ppa.launchpadcontent.net/lutris-team/lutris/ubuntu jammy InRelease
Hit:6 http://us.archive.ubuntu.com/ubuntu jammy InRelease
Hit:7 https://ppa.launchpadcontent.net/wfg/0ad/ubuntu jammy InRelease
Hit:8 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:9 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:10 http://us.archive.ubuntu.com/ubuntu jammy-security InRelease
Hit:11 https://regolith-desktop.org/release-ubuntu-jammy-amd64 jammy InRelease
Hit:12 https://packages.microsoft.com/repos/edge stable InRelease
Get:13 https://packages.microsoft.com/repos/ms-teams stable InRelease [17,5 kB]
Hit:14 https://packagecloud.io/slacktechnologies/slack/debian jessie InRelease
Fetched 66,3 kB in 4s (16,3 kB/s)
Reading package lists...
W: https://packagecloud.io/slacktechnologies/slack/debian/dists/jessie/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
  [+] Updating /var/cache/deb-get/deb-get.json
  [+] Updating /var/cache/deb-get/flameshot.json
  [+] Updating /var/cache/deb-get/pandoc.json
  [+] Updating /var/cache/deb-get/rclone.json
  [+] Updating /var/cache/deb-get/rpi-imager.json

**Comment by LyzardKing** [comment]
This is already fixed in main
You can run from the github repo, or wait for the next release.

**Comment by philclifford** [comment]
Just confirming main is seeing 2022.12.221341 which the above resolves to  "now" : 

```
deb-get show lens
  Lens
  Package:	lens
  Repository:	01-main
  Updater:	deb-get
  Installed:	No
  Published:	2022.12.221341
  Architecture:	amd64
  Download:	https://downloads.k8slens.dev/ide/Lens-2022.12.221341-latest.amd64.deb
  Website:	https://k8slens.dev/
  Summary:	The way the world runs Kubernetes.
```
